### PR TITLE
[1.3.7-prepare][Improvement] Impove text description, OSUtilsTest and reduce heap memory for create/upgrade-dolphinscheduler.sh

### PR DESCRIPTION
**Pull request by chengshiwen**
[1.3.7-prepare][Improvement] Impove text description, OSUtilsTest and reduce heap memory for create/upgrade-dolphinscheduler.sh, syncing with the latest dev

### CODE REVIEW

File: dolphinscheduler-common/src/test/java/org/apache/dolphinscheduler/common/os/OSUtilsTest.java [hunk 1]
```diff
@@ -37,39 +33,32 @@
...
+        double totalPhysicalMemorySize = OSUtils.totalPhysicalMemorySize();
+        logger.info("availablePhysicalMemorySize : {}", availablePhysicalMemorySize);
+        logger.info("totalPhysicalMemorySize : {}", totalPhysicalMemorySize);
+        Assert.assertTrue(availablePhysicalMemorySize >= 0.0);
+        Assert.assertTrue(totalPhysicalMemorySize >= 0.0);
     }
 
-
     @Test
     public void loadAverage() {
```
Comment (Technoboy-) [inline dolphinscheduler-common/src/test/java/org/apache/dolphinscheduler/common/os/OSUtilsTest.java]: Test method should named testXXX().

File: dolphinscheduler-common/src/test/java/org/apache/dolphinscheduler/common/os/OSUtilsTest.java [hunk 2]
```diff
@@ -37,39 +33,32 @@
...
+        double memoryUsage = OSUtils.memoryUsage();
+        logger.info("memoryUsage : {}", memoryUsage);
+        Assert.assertTrue(memoryUsage >= 0.0);
     }
 
     @Test
-    public void availablePhysicalMemorySize() {
-        logger.info("availablePhysicalMemorySize : {}", OSUtils.availablePhysicalMemorySize());
-        logger.info("availablePhysicalMemorySize : {}", OSUtils.totalMemorySize() / 10);
+    public void physicalMemorySize() {
```
Comment (Technoboy-) [inline dolphinscheduler-common/src/test/java/org/apache/dolphinscheduler/common/os/OSUtilsTest.java]: Could you change the method to testPhysicalMemorySize()?

File: dolphinscheduler-common/src/test/java/org/apache/dolphinscheduler/common/os/OSUtilsTest.java [hunk 3]
```diff
@@ -37,39 +33,32 @@
 
     @Test
     public void memoryUsage() {
```
Comment (Technoboy-) [inline dolphinscheduler-common/src/test/java/org/apache/dolphinscheduler/common/os/OSUtilsTest.java]: Also here

**Review by Technoboy- — CHANGES_REQUESTED** [review]

File: dolphinscheduler-common/src/test/java/org/apache/dolphinscheduler/common/os/OSUtilsTest.java (hunk 1 above)
Comment (chengshiwen) [reply dolphinscheduler-common/src/test/java/org/apache/dolphinscheduler/common/os/OSUtilsTest.java]: Done

File: dolphinscheduler-common/src/test/java/org/apache/dolphinscheduler/common/os/OSUtilsTest.java (hunk 3 above)
Comment (chengshiwen) [reply dolphinscheduler-common/src/test/java/org/apache/dolphinscheduler/common/os/OSUtilsTest.java]: Done

File: dolphinscheduler-common/src/test/java/org/apache/dolphinscheduler/common/os/OSUtilsTest.java (hunk 2 above)
Comment (chengshiwen) [reply dolphinscheduler-common/src/test/java/org/apache/dolphinscheduler/common/os/OSUtilsTest.java]: Done

**Review by Technoboy- — APPROVED** [review]
LGTM

**Pull request merged**